gravitee-am-common/src/main/java/io/gravitee/am/common/utils/ConstantKeys.java

```diff
@@ -93,6 +94,7 @@ public interface ConstantKeys {
     String PASSWORDLESS_CHALLENGE_KEY = "challenge";
     String PASSWORDLESS_CHALLENGE_USER_ID = "passwordlessUserId";
     String PASSWORDLESS_CHALLENGE_USERNAME_KEY = "passwordlessUsername";
+    String PASSWORDLESS_ORIGIN = "webAuthnSettingsOrigin";
```

Comment:
passwordlessOrigin to be consistent with the other fields

Reply:
done